blockchain/validate.go

```diff
@@ -1126,6 +1147,46 @@ func (b *BlockChain) checkConnectBlock(node *blockNode, block *btcutil.Block, vi
 		scriptFlags |= txscript.ScriptVerifyCheckLockTimeVerify
 	}
 
+	// Enforce CHECKSEQUENCEVERIFY during all block validation checks once
+	// the soft-fork deployment is fully active.
+	csvState, err := b.deploymentState(node, chaincfg.DeploymentCSV)
```

Comment:
I believe this should be the previous node since `deploymentState` calculates the state for the block _after_ the passed node.